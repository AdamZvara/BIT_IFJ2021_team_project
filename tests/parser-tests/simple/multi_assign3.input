require "ifj21"

global foo : function() : number, integer, string

function bar(i : integer, n : number) : number, integer
	write(i, " ",n,"\n")
	local s : string = ""
	n,i, s = foo()
	write(s)
	return n * 2.21,i

end

function foo() : number, integer, string
	local s : string = "DLOUHESLOVO"
	local n : number = 2.0*1.26
	local i : integer = #s

	return n,i,s
end

function main()
	bar(2,2)
end

main()
