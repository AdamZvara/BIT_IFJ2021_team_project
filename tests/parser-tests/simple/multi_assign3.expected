2 2
DLOUHESLOVO